scripts/uniqify-module-names.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+import json
+import argparse
+import shutil
+import os
+import sh
```

Comment:
This seems like it's no longer used, so you could also remove the addition from `conda-reqs/chipyard.yaml`.